### PR TITLE
Copy items to clipboard in Excel friendly format

### PR DESCRIPTION
**Pull request by terencesoh**
Change the Ctrl/Cmd-C clipboard copying of items to tab-delimited single line format. And add a header row as well. Makes it easier to paste in table format.

Limit the filter text in status bar to ~100 chars, so it doesn't cover the whole bar and hide other messages.

### CODE REVIEW

File: src/logtab.cpp [hunk 1]
```diff
@@ -905,8 +917,13 @@ void LogTab::UpdateStatusBar()
...
             }
             status += highlightOpt.m_value;
+            if (status.size() > 100)
+            {
+                status += "...";
+                break;
+            }
         }
-        status += "}";
+        status += "}  ";
```
Comment (luiseeo) [inline src/logtab.cpp]: what's the reason behind those extra white spaces?

File: src/logtab.cpp [hunk 2]
```diff
@@ -608,7 +620,7 @@ void LogTab::CopyItemDetails(const QModelIndexList& idxList)
                 }
                 else
                 {
-                    copyText += "N/A\t";
+                    copyText += "-\t";
```
Comment (luiseeo) [inline src/logtab.cpp]: is "-" better than just empty?

Comment (terencesoh) [inline src/logtab.cpp]: Two reasons for using "-":
- Consistency: This is mainly used for the calculated Elapsed column.  Other fields in our JSON logs will have "-" set when there is no value.
- Easier to identify the empty fields when you paste it in plain text editor.

File: src/logtab.cpp (hunk 1 above)
Comment (terencesoh) [inline src/logtab.cpp]: The extra white spaces is for padding, so the text doesn't stick right to the border.

Comment (luiseeo) [inline src/logtab.cpp]: I see (and now I can't unsee it hehe). I think we should handle the padding through the UI though, not by adding artificial spaces to the strings.

File: src/logtab.cpp (hunk 2 above)
Comment (luiseeo) [inline src/logtab.cpp]: 👍

File: src/logtab.cpp (hunk 1 above)
Comment (terencesoh) [inline src/logtab.cpp]: Changed the StatusBar class to set the label right margin.

**Pull request merged**